src/main/java/com/kodilla/ecommercee/dto/OrderDto.java

```diff
@@ -11,5 +13,7 @@
     private String status;
     private LocalDateTime dateTime;
     private String shippingAddress;
+    private User user;
+    private Cart cart;
 
```

Comment:
Myślę, że w klasie DTO powinny być pola UserDto i CartDto. Ja u siebie jako zaślepkę wpisałem klasę Cart tak jak Ty teraz, ale to dlatego, że chciałem Ci jak najmniej mieszać w Twoich klasach.
Dzięki temu również (jeśli np CartDto u mnie przechowuje tylko pole Long orderId, a nie Order order tak jak sama encja - unikniesz zapętlenie i błędu StackOverFlowException, który rozwiązałeś dodatkowymi getterami z adnotacjami @JsonBackReference (nie wiem, czy takie rozwiązanie jest lepsze/gorsze lub czy pasuje do pozostałej struktury kodu całego projektu - to już @danieloginski pewnie się odniesie ;) )